views/AdminLogs.html

```diff
@@ -22,6 +22,21 @@
     </button>
     <hr/>
 
+    <div class="row form-group">
+        <div class="col-xs-12">
+            <button id="SelectAll" class="btn btn-default" title="Select All" onclick="selectAllLogs()">
```

Comment:
Admin logs doesn't require a select or an ignore. It's just the full dump of the logs.